rs/state_manager/src/tree_hash.rs

```diff
@@ -165,6 +167,8 @@ mod tests {
     #[test]
     fn test_backward_compatibility() {
         fn state_fixture(certification_version: CertificationVersion) -> ReplicatedState {
+            assert!(certification_version <= CURRENT_CERTIFICATION_VERSION);
```

Comment:
Same as above: is there any benefit to this check that I'm missing?

Reply:
See above.